doc/manual/manual.tex

```diff
@@ -2625,118 +2625,56 @@ \subsubsection{Running \aspect{} models}
 \aspect{} itself. This installation procedure guarantees fastest runtimes, and largest flexibility,
-but might reveal incompatibilities with existing system libraries.
-
-\subsubsection{System prerequisites}
-
-In order to install \aspect{} and its dependencies, you should
-have your system set up to be able to compile and link programs. Additionally,
-\aspect{} needs a number of widely used libraries that are available
-for most operating systems. Therefore, you will need compilers for C, C++ and
-Fortran, the GNU make system, the CMake build system, and the libraries and
-header files of BLAS, LAPACK and zlib, which is used for compressing
-the output data. To use more than one process for your computations
-you will need to install a MPI library, its headers and the
-necessary executables to run MPI programs. There are some optional packages
-for additional features, like the HDF5 libraries for additional output formats,
-PETSC for alternative solvers, and Numdiff for checking \aspect{}'s test
-results with reasonable accuracy, but these are not strictly required, and in
-some operating systems they are not available as packages but need to be
-compiled from scratch.
-Finally, for obtaining a recent development version of \aspect{} you will
-need the git version control system.
-
-An exemplary command to obtain all required packages on Ubuntu 14.04 would be:
-\begin{verbatim}
-sudo apt-get install build-essential \
-                     cmake \
-                     gcc \
-                     g++ \
-                     gfortran \
-                     git \
-                     libblas-dev \
-                     liblapack-dev \
-                     libopenmpi-dev \
-                     numdiff \
-                     openmpi-bin \
-                     zlib1g-dev
-\end{verbatim}
-
-\subsubsection{Software prerequisites}
+but usually requires more work than the options mentioned in the previous sections. 
+While it is possible to install ASPECT's dependencies in particular \pfrst{}, \trilinos{}, 
+and \dealii{} manually, we recommend to use the 
+\texttt{candi} software (see \url{https://github.com/dealii/candi}). \texttt{candi} was written
+as an installation program for deal.II, and includes a number of system specific instructions
+that will be listed when starting the program. It can be flexibly configured to allow for
+non-default compilers or libraries (e.g. Intel's MKL instead of LAPACK) by changing entries
+in the configuration file \texttt{candi.cfg}, or by providing platform specific installation files.
+
+In its default configuration it downloads and
+compiles a \dealii{} configuration that is able to run \aspect, but it also contains a number of
+packages that are not required (and that can be safely disabled if problems occur during the
+installation). We require at least the packages \pfrst{}, one of either \trilinos{} or \texttt{petsc},
```

Comment:
maybe say "..., \trilinos{} (or alternatively, if you want to experiment, \texttt{PETSc})" to make clear that almost everyone is actually using the former.